packages/mobile/src/inHouseLiquidity/index.ts

```diff
@@ -0,0 +1,42 @@
+import networkConfig from 'src/geth/networkConfig'
+import { serializeSignature, signMessage } from '@celo/utils/lib/signatureUtils'
+import { getStoredMnemonic } from 'src/backup/utils'
+import { generateKeys } from '@celo/utils/lib/account'
+
+export const createPersonaAccount = async (accountMTWAddress: string): Promise<Response> => {
+  const message = `post /account/create ${JSON.stringify({ accountMTWAddress })}`
```

Comment:
the Date header needs to be in the signature too. see packages/mobile/src/inHouseLiquidity/index.test.ts